HomeTask8/src/main/webapp/WEB-INF/add/addGroup.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Add Group</title>
+</head>
+<body>
+<form method="post" action="/addGroup">
+    <div>
+        <h2>Add group.</h2>
+    </div>
+    <div>
+        <p>Title</p><input type="text" name="title">
+        <button type="submit">Add</button>
+    </div>
+    <a href="/university">Return to back</a>
```

Comment:
It's better to move this link a bit below, after the `form` as there is no sense to place it inside